packages/edit-site/src/components/navigation-sidebar/navigation-panel/menus/content-posts.js

```diff
@@ -61,7 +61,6 @@ export default function ContentPostsMenu() {
 			type: 'page',
 			path: '/',
 			context: {
-				query: { categoryIds: [] },
```

Comment:
This change is also expected? It's the only place where I have doubts. Otherwise, great cleanup 👍🏻

Reply:
Yes. This was a leftover from the first FSE template navigation that tried to have a way for QueryLoop to live outside the Query. We have `inherit` option now.

Reply:
Awesome, thanks for confirmation 👍🏻